src/routes/index.tsx

```diff
@@ -11,6 +12,12 @@ export const paths = {
   root: '/',
   signIn: '/sign-in',
   survey: '/surveys/:id',
+  question: '/surveys/:id/questions',
+};
+
+export const questionPath = (): string => {
+  const questionPaths = paths.question.split('/');
+  return questionPaths[questionPaths.length - 1];
```

Comment:
Can you explain this? I'm not sure why do we need this function 🤔

Reply:
@hoangmirs I want to extract the `questions` part but it seems that I may have overcomplicated it. Just create a constant is enough 😅 Will update now

Reply:
@hoangmirs fixed in [cf449dd](https://github.com/manh-t/react-survey/pull/75/commits/cf449ddc877e55d3fd751f3393ec3d644c023942)